www/index.html

```diff
@@ -3,7 +3,7 @@
   <head>
     <meta charset="utf-8">
     <meta name="viewport" content="initial-scale=1, maximum-scale=1, user-scalable=no, width=device-width">
-    <meta http-equiv="Content-Security-Policy" content="default-src 'self' data: gap: https://ssl.gstatic.com http://nominatim.openstreetmap.org https://e-mission.eecs.berkeley.edu https://berkeley.qualtrics.com https://jfe-cdn.qualtrics.com emission:;
+    <meta http-equiv="Content-Security-Policy" content="default-src 'self' data: gap: https://ssl.gstatic.com http://nominatim.openstreetmap.org https://e-mission.eecs.berkeley.edu https://berkeley.qualtrics.com https://jfe-cdn.qualtrics.com emission: https://em-game.eecs.berkeley.edu;
```

Comment:
Do your really need this?
https://github.com/apache/cordova-plugin-inappbrowser#urls-that-are-not-white-listed

> The InAppBrowser is not subject to the whitelist, nor is opening links in the system browser.